src/abstract/array_with_ids.js

```diff
@@ -30,31 +31,38 @@ export class ArrayWithIds {
      * @param {Function} fn - The function to be applied to each array element.
      */
     mapArrayInPlace(fn) {
-        if (!_.isFunction(fn)) throw new Error("ArrayWithIds.mapArray: must pass function as argument");
+        if (!_.isFunction(fn))
+            throw new Error('ArrayWithIds.mapArray: must pass function as argument');
         this.array = this.array.map((...args) => fn(...args));
     }
 
-    getArrayElementByIndex(idx) {return this.array[idx]}
+    getArrayElementByIndex(idx) {
```

Comment:
Maybe we can configure the linter to keep one-line functions?

Reply:
We can change it to:
```
getArrayElementByIndex = (idx) => this.array[idx];
```

Reply:
Sounds good